### PR TITLE
Merge branch 'release/2.1' into dev

### PR DESCRIPTION
**Pull request by natemcmaster**

### CODE REVIEW

**Comment by natemcmaster** [comment]
The dev branch is building versions older than what is on release/2.1

File: version.props [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <VersionSuffix Condition="'$(VersionSuffix)' != '' And '$(FeatureBranchVersionSuffix)' != ''">$(FeatureBranchVersionPrefix)$(VersionSuffix)-$([System.Text.RegularExpressions.Regex]::Replace('$(FeatureBranchVersionSuffix)', '[^\w-]', '-'))</VersionSuffix>
     <VersionSuffix Condition="'$(VersionSuffix)' != '' And '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
 
-    <ExperimentalVersionPrefix>0.1.0</ExperimentalVersionPrefix>
-    <ExperimentalVersionSuffix>preview3</ExperimentalVersionSuffix>
+    <ExperimentalVersionPrefix>0.2.0</ExperimentalVersionPrefix>
```
Comment (smitpatel) [inline version.props]: Does this matter?
It is used in Cosmos.Sql package only, which is not being shipped. And we don't even have a versioning story for it yet.

Comment (natemcmaster) [reply version.props]: Yes. Packages from release/2.1 and dev end up on the same myget feeds

Comment (smitpatel) [reply version.props]: The package contains no source code. 😆

**Comment by natemcmaster** [comment]
Ok. Since nuget push doesn't overwrite by default, let's go with this version bump, just to keep dev moving along smoothly.

**Pull request merged**